app/assets/javascripts/provider/admin/cms/sidebar.js.coffee

```diff
@@ -336,113 +349,100 @@ class SidebarFilter
-            }
-          </style>
-
-          <ul>
-            <li class="cms-section" data-behavior="toggle search">
-              <span id="cms-sidebar-collapse-all" class="collapse-button"><i class="folder-open-o fa-fw"></i></span>
-              <%= link_to(section.title, section.edit_path) %>
-              <%= render(section) %>
-            </li>
-            <li class="no-results">No Content Found</li>
-          </ul>
-          """
-  @content = """
-          <ul id="cms-sidebar-section-<%= section.id %>">
-            <% _.each(pages, function(page) { %>
-              <li class="cms-page" data-id="<%= page.id %>" data-behavior="drag search" data-param="cms_page" <%= search(page) %>>
-                <%= icon_link_to('file', page.title, page.edit_path, [ page.path, '(' + page.title + ')' ]) %>
-              </li>
-            <% }); %>
-
-            <% _.each(files, function(file) { %>
-              <li class="cms-file" data-id="<%= file.id %>" data-behavior="drag search" data-param="cms_file" <%= search(file) %>>
-                <%= icon_link_to('paperclip', file.attachment_file_name, file.edit_path) %>
-              </li>
-            <% }); %>
-
-            <% _.each(builtins, function(builtin) { %>
-              <li class="cms-builtin" data-id="<%= builtin.id %>" data-behavior="drag search" data-param="cms_builtin" <%= search(builtin) %>>
-                <%= icon_link_to('cog', builtin.title, builtin.edit_path) %>
-              </li>
-            <% }); %>
-
-            <% _.each(sections, function(section){ %>
-              <li class="cms-section"  data-id="<%= section.id %>" data-behavior="toggle drag search" data-param="cms_section" <%= search(section) %>>
-                <%= icon('folder-open fa-fw') %>
-                <%= link_to(section.title, section.edit_path) %>
-                <%= render(section) %>
-              </li>
-            <% });  %>
-          </ul>
-             """
-  @layouts = """
-             <h3>Layouts</h3>
-             <ul>
-              <% _.each(layouts, function(layout) { %>
-                <li <%= search(layout) %> data-behavior="search">
-                  <%= icon_link_to('code', layout.title, layout.edit_path) %>
-                 </li>
-              <% }); %>
-              <% if(_.isEmpty(layouts)) { %><li data-behavior="search">No Layouts</li><% }; %>
-              <li class="no-results">No Layouts Found</li>
-             </ul>
-             """
-
-  @portlets = """
-              <h3>Portlets</h3>
-              <ul>
-                <% _.each(portlets, function(portlet) { %>
-                  <li <%= search(portlet) %>>
-                    <%= icon_link_to('rocket', portlet.title, portlet.edit_path) %>
-                  </li>
-                <% }); %>
-                <% if(_.isEmpty(portlets)) { %><li data-behavior="search">No Portlets</li><% }; %>
-                <li class="no-results">No Portlets Found</li>
+  @root: ({ section, render }) ->
```

Comment:
In which screen is this used? I'd like to verify it looks like before. Because you are removing some styling here.

Reply:
This renders the CMS sidebar.

Reply:
Thanks!